cvat-ui/src/components/annotation-page/styles.scss

```diff
@@ -238,6 +238,77 @@
     }
 }
 
+.cvat-canvas-image-setups-trigger {
+    position: absolute;
+    background: $background-color-2;
+    bottom: 0;
+    left: 50%;
+    opacity: 0.5;
+    border-radius: 6px 6px 0 0;
+    border: 1px solid $border-color-3;
+    z-index: 100;
+    padding: 4px 12px;
```

Comment:
Just a question, should we use `$grid-unit-size` here?

Reply:
I am not sure because sometimes grid-unit-size (8px) looks too much in paddings, border radios, etc.